Scripts/Python/ki/xKIChat.py

```diff
@@ -1452,7 +1452,7 @@ def RollDice(self, dice_str):
             return
 
         # Parse common dice notation
-        dice_opt = re.match("^(\d+)d(\d+)$", dice_str)
+        dice_opt = re.match("^(\\d+)d(\\d+)$", dice_str)
```

Comment:
Perhaps an even better fix would be to use raw string literals:
```suggestion
        dice_opt = re.match(r"^(\d+)d(\d+)$", dice_str)
```

Reply:
I saw raw strings as an alternate option in my researching, but I was familiar with using \\ in other languages so I favored this option. It looked like \\s was being used in a couple places in xKIHelpers.py as well. Is there a reason why a raw string would be better than using \\ escapes?

Reply:
We generally prefer raw strings instead of manually escaping backslashes due to the clarity it offers. This way, we're looking at the exact regex instead of having to go through the cognitive load of decoding the regex any time we want to tweak it. It's nice to be able to copy and paste directly from regex101.com, for example. The only time we'd probably want to *not* use raw strings is if there is a need to use specific unicode codepoints.

Reply:
Ok, I will update those places. Did you want me to update the uses in xKIHelpers.py as well to use raw strings?

Reply:
If those are also raising DeprecationWarnings, that would be great!

Reply:
They are actually using the \\\\s version already, so they don't throw any warnings. But if we wanted to keep things more consistent?

Reply:
It would be a welcome change 😄

Reply:
Adjustments complete to use raw strings in the 2 deprecation warning locations and the 2 locations that were using \\\\s